### DISCUSSION TITLE
Penwidth

### DISCUSSION THREAD

**Discussion by bit-of-a-shambles** [monora/rgl#61] [Ideas]
Would it be possible to add this attribute?

https://graphviz.org/docs/attrs/penwidth/

**Comment by monora** [comment]
> Would it be possible to add this attribute?

I guess yes. See [lib/rgl/rdot.rb](https://www.rubydoc.info/github/monora/rgl/RGL/DOT#NODE_OPTS-constant) . This refers to an old version of the documentation from 2008.

The new version http://www.graphviz.org/pdf/dotguide.pdf from 2015 includes _penwidth_ and many others. Would you mind creating an issue to update to the new version? And even better create a pull request...
